### DISCUSSION TITLE
map duplicating when updated

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#19342] [Q&A] [answered]
mapset in question: https://osu.ppy.sh/beatmapsets/17819

when updating the mentioned beatmap, the mapset duplicates itself on the carousel while also failing to actually update  

[updater.log](https://github.com/ppy/osu/files/9174894/updater.log)
[runtime.log](https://github.com/ppy/osu/files/9174895/runtime.log)
[performance-update.log](https://github.com/ppy/osu/files/9174896/performance-update.log)
[performance-input.log](https://github.com/ppy/osu/files/9174897/performance-input.log)
[performance-draw.log](https://github.com/ppy/osu/files/9174898/performance-draw.log)
[performance-audio.log](https://github.com/ppy/osu/files/9174899/performance-audio.log)
[performance.log](https://github.com/ppy/osu/files/9174900/performance.log)
[network.log](https://github.com/ppy/osu/files/9174901/network.log)
[database.log](https://github.com/ppy/osu/files/9174902/database.log)

picture of this happening:
![osu_2022-07-23_19-45-32](https://user-images.githubusercontent.com/44514667/180627564-6da1913e-0bad-4b41-92e0-c0a828694c83.png)

version: 2022.723.0

**Comment by frenzibyte** [comment]
The importer relies on the online ID that's included within the `.osu` file to mark the older version for removal. On older beatmaps (I persume), there are no IDs included, thus the older beatmap remains each time you update.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/19369
